pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.mypy]
+files = "inhouse_bot/database_orm/tables/*.py"

```

Comment:
Over time, more files will be included in the files to check. Starting with the table files for now.